### PR TITLE
removed checkout of dataframes and dataarray in runtests.jl

### PR DESCRIPTION
**Pull request by jeff-regier**
@andreasnoack says it's no longer necessary

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-julia 0.6-
+julia 0.6
```
Comment (andreasnoack) [inline REQUIRE:1]: I believe this is too early. I'm pretty sure Julia master won't satisfy this requirement.

Comment (jeff-regier) [reply REQUIRE:1]: It seems to. I didn't get a warning.

Comment (jeff-regier) [reply REQUIRE:1]: I'm using `Version 0.6.0-pre.alpha.313 `

Comment (jeff-regier) [reply REQUIRE:1]: Oh, you're right. I changed in back.

**Pull request merged**